src/insert.rs

```diff
@@ -189,7 +161,9 @@ impl<T> Insert<T> {
     where
         T: Serialize,
     {
-        assert!(self.sender.is_some(), "write() after error");
```

Comment:
It's important check detecting incorrectly written code by the user. Now after errors you quietly start a new `INSERT` statement that most likely not what the user wanted

Reply:
Should be resolved now.